.github/workflows/copy-react-demo-s3.yml

```diff
@@ -57,11 +57,11 @@ jobs:
 
       - name: Build React Demo staging
         if: ${{ github.ref_name != 'main' }}
-        run: yarn build:react:video-demo --mode staging
+        run: yarn build:react:video-demo
```

Comment:
The gleap api key depends on the mode set to "staging". Gleap will only be visible on staging.